src/app/api/slides/route.ts

```
@@ -129,7 +129,17 @@ export async function POST(request: NextRequest) {
-        const narrateResult = await slideNarrator.narrateSlide();
+        if (!slideFile || !language) {
+          return NextResponse.json(
+            { error: 'Slide file and language required for narration' },
+            { status: 400 }
+          );
+        }
+
+        // Ensure narration is loaded before attempting to narrate
+        await slideNarrator.loadNarration(slideFile, language);
+        
+        const narrateResult = await slideNarrator.narrateSlide(slideNumber);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)